app/views/users/new.html.haml

```diff
@@ -1,28 +1,18 @@
-%h1 Sign Up
-%br
-= form_for @user do |f|
-  = f.label :first_name
-  %br
-  = f.text_field :first_name
-  %br
-  = f.label :last_name
-  %br
-  = f.text_field :last_name
-  %br
-  = f.label :email
-  %br
-  = f.text_field :email
-  %br
-  = f.label :phone_number
-  %br
-  = f.telephone_field :phone_number
-  %br
-  = f.label :password
-  %br
-  = f.password_field :password
-  %br
-  = f.label :password_confirmation
-  %br
-  = f.password_field :password_confirmation
-  %br
-  = f.submit 'Sign Up'
+%section.account_form  
+  %h1 Create your Pad Crash Account
+  %br
```

Comment:
In the future, we can probably achieve this spacing using the flex box properties.